packages/create-fuse-app/src/index.ts

```diff
@@ -233,16 +233,20 @@ const updateTSConfig = async (targetDir: string) => {
     ) {
       const updatedTsConfig = {
         ...tsConfig,
+        include: [
+          ...(tsConfig.include || []),
+          hasSrcDir ? './src/fuse/graphql.ts' : './fuse/graphql.ts',
+        ],
         compilerOptions: {
           ...tsConfig.compilerOptions,
           plugins: [
             ...(tsConfig.compilerOptions?.plugins || []),
             {
               name: '@0no-co/graphqlsp',
               schema: './schema.graphql',
-              disableTypegen: true,
-              templateIsCallExpression: true,
-              template: 'graphql',
+              tadaOutputLocation: hasSrcDir
```

Comment:
This would mean that we rely on the LSP to output the `introspection.ts` and will generate `graphql.ts` once as ewll